docs/manual/guides/local-installation/ddev.de.md

```diff
@@ -26,96 +26,69 @@ mehr Informationen zur Installation dieser Programme durchlesen.
-
-sudo apt update && sudo apt install -y ddev
+ddev config --project-type=php --docroot=public --webserver-type=apache-fpm --php-version=8.2 --create-docroot --timezone=Europe/Berlin
 ```
 
-Evtl. nach der Installation noch die lokalen SSL-Zertifikate installieren (im Anschluss Browser neu starten).
+Contao 5.3 installieren:
 
 ```shell
-mkcert -install
+ddev composer create contao/managed-edition:5.3
 ```
 
-Installation updaten
+Nach der Installation müssen die Datenbankzugangsdaten in die `.env.local` eingetragen werden. In diesem Zug richten wir auch direkt Mailpit ein.
 
-```shell
-sudo apt update && sudo apt upgrade
+```env
+APP_ENV=prod
+DATABASE_URL=mysql://db:db@db:3306/db
+MAILER_DSN=smtp://localhost:1025
 ```
 
-
-## Projekt erstellen
-
-Öffne die Konsole deiner Wahl, erstelle das gewünschte Verzeichnis und wechsle danach in ebendieses.
+Im Anschluss die Datenbank anlegen:
 
 ```shell
-mkdir -p  ~/Projekte/contao/contao-ddev && cd ~/Projekte/contao/contao-ddev
+ddev exec "bin/console contao:migrate"
 ```
 
-DDEV-Konfiguration anlegen mit:
+Backend-User anlegen:
 
 ```shell
-ddev config
+ddev exec "bin/console contao:user:create"
 ```
 
-DDEV-Einstellungen vornehmen, als __Project Type__ auf jeden fall `php` auswählen. Die __Docroot Location__ erstmal leer lassen, da es bei Neuinstallationen noch keinen `public` Ordner gibt und DDEV dann nicht starten kann.
+Projekt im Browser aufrufen:
 
 ```shell
-ddev start
+ddev launch
 ```
 
-Zur Installation via Konsole ist es am einfachsten, sich via SSH mit dem Container zu verbinden.
+{{% notice note %}}
 
-```shell
-ddev ssh
-```
+Mit `ddev launch contao` kommst du direkt zur Administration.
 
-```shell
-composer create-project contao/managed-edition contao 4.13
-```
+{{% /notice %}}
 
-In der `.ddev/config.yml` nun das Docroot anpassen und ddev neu starten.
+## Zusatz Informationen
 
-```yml
-docroot: "contao/public"
-```
+`ddev start` startet das Projekt, `ddev stop` beendet es. Stelle vorher sicher, dass du in den Projektordner gewechselt hast.
 
-Um Apache anstatt NGINX zu verwenden, den Eintrag `webserver_type: nginx-fpm` in `apache-fpm` ändern.
+`ddev poweroff` kann aus jedem Verzeichnis heraus alle gestarteten Projekte/Container stoppen.
 
-```yml
-webserver_type: apache-fpm
-```
+Mit `ddev ssh` wechselst du in die Shell des Containers und kannst auf der Konsole arbeiten. Die `ddev` Binary steht im Container nicht zur Verfügung, also erst mit `exit` auf die Host-Konsole wechseln.
 
-Die `ddev` Binary steht im Container nicht zur Verfügung, also erst mit `exit` auf die Host-Konsole wechseln.
+`ddev describe` gibt eine Übersicht der Services, die im Projekt zur Verfügung stehen und wie du sie erreichst.
 
-```shell
-ddev restart
-```
+`ddev xdebug on` startet Xdedug. [Informationen zum IDE-Setup](https://ddev.readthedocs.io/en/latest/users/debugging-profiling/step-debugging/#ide-setup)
 
 Eine Datenbank gibt es schon in DDEV. Die Verbindungsdaten für die Installation lauten:
```

Comment:
I think this could be removed here (see: DATABASE_URL=mysql://db:db@db:3306/db )